test/e2e/cli.go

```diff
@@ -87,6 +145,17 @@ func NewKubectl(namespace string) Kubectl {
 	}
 }
 
+//Create will help you to create a resource in any namespace
+func (k Kubectl) Create(args ...string) *icmd.Result {
+	if k.namespace != "" {
+		args = append(args, "--namespace", k.namespace)
+	}
+	cmd := append([]string{"kubectl", "create", "-f"}, args...)
+	res := icmd.RunCmd(icmd.Cmd{Command: cmd, Timeout: tknTimeout})
```

Comment:
Can `tknTimeout` be renamed to`timeout` since it's being used for `kubectl` in this case?

Reply:
this makes sense I will update it.